app/assets/v2/js/cart.js

```diff
@@ -497,6 +437,76 @@ Vue.component('grants-cart', {
   },
 
   methods: {
+    // Array of objects containing all donations and associated data
+    computeDonationInputs(destGitcoinAddress) {
+      let isPolygon = destGitcoinAddress == gitcoinAddressPolygon;
+
+      if (!this.grantsByTenant || this.tabSelected !== 'ETH') {
+        return undefined;
+      }
+
+      // Generate array of objects containing donation info from cart
+      const donations = this.grantsByTenant.map((grant, index) => {
```

Comment:
it doesn't look like the `index` variable is being used, we can remove it from the params list.